README.md

```diff
@@ -364,7 +365,7 @@ set up PersistentVolume and PersistentVolumeClaim visit [GKE Filestore documenta
 
 Creating Filestore storage and attaching it to workload can be achieved in two steps:
 
-* Create a simple Storage, that attaches existing filestore instance to your workloads. User must specify `--type=gcpfilestore`.
+* Create a simple Storage, that attaches existing filestore instance to your workloads. User must specify `--type=gcpfilestore`. This command will create a new instance of Filestore, which you can find in your gcp console, or by running `here is the command for listing filestore instance`
 
     ```shell
     python3 xpk.py storage create fs-storage-attach --project=$PROJECT
```

Comment:
Should this be changed to attach? If not some arguments like --manifest should be removed and other like --size and --tier should be added.